apps-rendering/src/components/InPageNewsletterSignup/NotSupportedMessage.tsx

```diff
@@ -0,0 +1,18 @@
+// ----- Imports ----- //
+
+import { InlineError } from '@guardian/source-react-components';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+const NotSupportedMessage: FC = () => {
+	return (
+		<InlineError>
+			Your version of the app does not support newsletter sign-ups.
```

Comment:
I wonder if we could improve this message for people seeing it. Some ideas I had, but probably require product input:

- mention that they can upgrade using their app store
- if they are unable to upgrade the app, mention they can subscribe to the newsletter using the website